website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+
+### Deprecated Debugging Features
+
+#### Flipper ↔ React Native integration
+
+We are gradually moving away from Flipper as the default tool for debugging React Native apps. In 0.73, this begins by deprecating the native Flipper integration included with React Native (bootstrap code which wires up Flipper's core plugins). We will be removing this integration and dependency in the next release — meaning parts of Flipper such as the Network plugin will cease to work.
+
+**Unchanged**: Flipper as a standalone product for native app debugging will continue to exist. Even after its removal from new React Native projects in future, developers will be able to [manually add Flipper to their app](https://fbflipper.com/docs/getting-started/android-native/) if they wish.
+
+For more information on why we are moving away from Flipper, [view the RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0641-decoupling-flipper-from-react-native-core.md).
+
+#### Remote JavaScript Debugging
+
+Remote JavaScript Debugging is a legacy debugging mode that connects an external web browser (Chrome) to your app and runs your JavaScript code inside a web page, i.e. `http://localhost:8081/debugger-ui`. This model could lead to inconsistent app behaviour while debugging, and is incompatible with native modules under the New Architecture.
+
+In 0.73, Remote JavaScript Debugging is deprecated and has been removed from the Dev Menu. Enabling the remote debugger must now be done manually via the `NativeDevSettings` API. Doing this is covered in the [Other Debugging Methods docs](/docs/next/other-debugging-methods#remote-js-debugging).
+
+:::info
+Remote JavaScript Debugging was previously the default debugging experience for apps using JavaScriptCore (JSC). We recommend [Safari Developer Tools (direct JSC debugging)](/docs/next/other-debugging-methods#safari-developer-tools-direct-jsc-debugging) instead, for iOS apps.
+
+We recommend using [Hermes](/docs/hermes) for a consistent debugging experience on all platforms.
+:::
+
+### New Architecture Updates
+
+We continue active development on the New Architecture and are fully invested in bringing it to the OSS community. You can read the latest updates in the [New Architecture working group](https://github.com/reactwg/react-native-new-architecture/discussions).
+
+## Breaking Changes
+
+### Babel Package Renames
+
+We've relocated two Babel-related packages out of Metro and into React Native's repository and versioning scheme. The new versions of these packages support New Architecture features in 0.73, meaning these dependencies must be updated.
+
+Please follow the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) when upgrading, to ensure you have updated these dependencies. If you were using `metro-react-native-babel-preset` in your Babel config file, don't forget to update that too.
```

Comment:
> don't forget to update that too.

Reading this i'd think "Update, how?!" and then have to read the next section. 

Something like:

> Please follow the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) when upgrading, to ensure you have updated these dependencies. If you were using `metro-react-native-babel-preset` in your Babel config file, you'll need to use the new package name.

or

> Please follow the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) when upgrading, to ensure you have updated these dependencies. Some packages have been renamed: